src/exporter.py

```diff
@@ -120,22 +121,26 @@ def track_queue_length(self):
 
         with self.app.connection() as connection:
             for queue in self.queue_cache:
-                try:
-                    ret = connection.default_channel.queue_declare(
-                        queue=queue, passive=True
+                if isinstance(connection.default_channel.client, Redis):
```

Comment:
I think it's better to use `type(connection.default_channel.client).__name__ == 'Redis'` to get rid of extra package dependency.

Reply:
redis-py is already in the dependencies of this project